flink-table/flink-table-api-java-bridge/pom.xml

```diff
@@ -52,5 +52,10 @@ under the License.
 			<artifactId>flink-streaming-java_${scala.binary.version}</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
```

Comment:
Add a comment that we can drop this dependency, once we have source/sink interfaces in `flink-table-common`.